README.md

```diff
@@ -44,8 +44,8 @@ sudo apt install clang-format cmake-curses-gui lcov doxygen
 First make sure you [have the developer tools installed](http://osxdaily.com/2014/02/12/install-command-line-tools-mac-os-x/) and then
 
 ```
-# Installing the bare build requirements
-brew install cmake
+# Installing the library
+brew install h3
```

Comment:
This is awesome, but I'm not sure this is the right part of the README - maybe we ought to put this up higher, under `Installing`? The current "Building from source" section is really more developer-focused, and developers will probably want to make sure they have `cmake` installed. The new formula makes installation (particularly of the command-line tools) much easier, so it should probably go higher up.